src/site/content/en/blog/image-support-for-async-clipboard/index.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Image support for the async clipboard API
+subhead: Starting in Chrome 76, the Asynchronous Clipboard API now handles some images, in addition to text.
+authors:
+  - thomassteiner
+description: Chrome 76 adds expands the functionality of the Async Clipboard API to add support for png images. Copying and pasting images to the clipboard has never been easier.
+date: 2019-07-03
+updated: 2019-10-08
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - capabilities
+  - chrome76
+  - cutandcopy
+  - execcommand
+  - input
+  - clipboard
+hero: hero.jpg
+alt: Clipboard with shopping list
+---
+
+Chrome 66, added support for the
+[text portion](https://developers.google.com/web/updates/2018/03/clipboardapi)
+of the [Asynchronous Clipboard API](https://w3c.github.io/clipboard-apis/#async-clipboard-api).
+Chrome 76 has added support for images makes it easy to programmatically
+copy and paste **png** images.
+
+{% Aside 'caution' %}
+  At the time of writing, only png files are supported. Support for other
+  images and file formats will be added in the future.
+{% endAside %}
+
+Before we dive in, let's take a brief look at how the Asynchronous Clipboard
+API works. If you remember the details, skip ahead to the [image section](#images).
+
+## Recap of the Asynchronous Clipboard API {: #recap }
+
+Before describing image support, I want to review how the Asynchronous Clipboard
+API works. Feel free to [skip ahead](#images) if you're already comfortable
+using the API.
```

Comment:
```suggestion
```

This paragraph repeats what the paragraph on line 32 just said

P.S. I'm not sure if the suggested edit will apply properly for all 3 lines. I've never tried it before.